public/modules/articles/articles.viz.js

```diff
@@ -499,6 +515,9 @@ var valign, stacked, articlesScope, aligned = true;
                                         .popover(function () {
                                             return section_hover(d, curS)
                                         })
+                                        .on("mouseenter", function() {
+                                            $(".popover").addClass("articles-popover");
+                                        })
                                         .text(clean_premier(titre_section(sub_section(curS), longlabel)));
                                     ct++;
```

Comment:
Clearly that's not ideal. We can't move the popover to the module container otherwise they will not be able to overflow it (that's the issue with {container:body} we were talking about). We cannot manipulate the popover div "at the beginning of the module load" because bootstrap generates them on mouseenter.

Anyway, I'll try to find something.